[2024-06-28T14:39:39.363+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T14:39:39.372+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T14:39:38.071375+00:00 [queued]>
[2024-06-28T14:39:39.377+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T14:39:38.071375+00:00 [queued]>
[2024-06-28T14:39:39.377+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T14:39:39.385+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): export_data_from_mysql> on 2024-06-28 14:39:38.071375+00:00
[2024-06-28T14:39:39.394+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=594) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-28T14:39:39.394+0000] {standard_task_runner.py:63} INFO - Started process 596 to run task
[2024-06-28T14:39:39.392+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_transfer_to_gcs', 'export_data_from_mysql', 'manual__2024-06-28T14:39:38.071375+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/to_gcs.py', '--cfg-path', '/tmp/tmpdhmky2g4']
[2024-06-28T14:39:39.395+0000] {standard_task_runner.py:91} INFO - Job 147: Subtask export_data_from_mysql
[2024-06-28T14:39:39.438+0000] {task_command.py:426} INFO - Running <TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T14:39:38.071375+00:00 [running]> on host 25c2a2aa63fd
[2024-06-28T14:39:39.522+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transfer_to_gcs' AIRFLOW_CTX_TASK_ID='export_data_from_mysql' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T14:39:38.071375+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T14:39:38.071375+00:00'
[2024-06-28T14:39:39.524+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T14:39:39.559+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2024-06-28T14:39:39.955+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/dags/data/exported_data2.csv
[2024-06-28T14:39:39.956+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T14:39:39.972+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_transfer_to_gcs, task_id=export_data_from_mysql, run_id=manual__2024-06-28T14:39:38.071375+00:00, execution_date=20240628T143938, start_date=20240628T143939, end_date=20240628T143939
[2024-06-28T14:39:40.009+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T14:39:40.043+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T14:39:40.045+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
